.github/workflows/push-tests.yml

```diff
@@ -128,38 +128,38 @@ jobs:
           # Build backend
           docker buildx build --cache-from=type=registry,ref=${{ secrets.ECR_REPO }}/genepi-backend:branch-trunk -t ${{ secrets.ECR_REPO }}/genepi-backend:latest src/backend
           make backend-check-style
-  e2e-test:
-    runs-on: ubuntu-20.04
-    steps:
-      - name: Configure AWS prod Credentials
-        uses: aws-actions/configure-aws-credentials@v1
-        with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: us-west-2
-          role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
-          role-duration-seconds: 900
-      - name: Login to ECR
-        uses: docker/login-action@v1
-        with:
-          registry: ${{ secrets.ECR_REPO }}
-      - uses: actions/checkout@v2
-      - name: Playwright tests
-        run: |
-          echo "DOCKER_REPO=${DOCKER_REPO}" > .env.ecr
-          # Build backend image
-          docker buildx build --cache-from=type=registry,ref=${{ secrets.ECR_REPO }}/genepi-backend:branch-trunk -t ${{ secrets.ECR_REPO }}/genepi-backend:latest src/backend
-          # Build frontend
-          docker buildx build --cache-from=type=registry,ref=${{ secrets.ECR_REPO }}/genepi-frontend:branch-trunk -t ${{ secrets.ECR_REPO }}/genepi-frontend:latest src/frontend
-          mkdir -p src/frontend/build
-          chmod a+rwx src/frontend/build # TODO temp hack - need a better plan for the build dir.
-          make local-init
-          echo "Init complete - testing web service"
-          # docker-compose exec -T frontend wget -T 10 --tries 20 localhost:8000/
-          echo "Webserver up, running tests!"
-          docker-compose logs -f frontend &
-          export S3_PREFIX=${{ secrets.CI_SCREENSHOT_PREFIX }}/${{ github.run_id }}/
-          make frontend-e2e-ci
+#  e2e-test:
```

Comment:
Nit: Probably good to drop a comment at the top just to say why this whole block has been commented out
```suggestion
# Test has been flaky for a couple months (~April 2022). Commenting out until we have
# some test engineering support to suss out the issue and clean things up.
#  e2e-test:
```